easybuild/easyconfigs/g/gffcompare/gffcompare-0.10.6-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'gffcompare'
+version = '0.10.6'
+
+homepage = 'https://github.com/gpertea/%(namelower)s'
+description = """gffcompare provides classification and reference annotation mapping and
+ matching statistics for RNA-Seq assemblies (transfrags) or other generic GFF/GTF files."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['http://ccb.jhu.edu/software/stringtie/dl/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['e75cc9fff42cea2d02752bd96e0e0451b268384247606e0e27193d15f4a54477']
+
+builddependencies = [('binutils', '2.30')]
+
+buildopts = " release"
+
+files_to_copy = ['%(namelower)s', 'LICENSE', 'README.md']
+
+modextrapaths = {'PATH': ''}
+
+sanity_check_paths = {
+    'files': ['%(namelower)s'],
+    'dirs': []
+}
+
+sanity_check_commands = ['%(namelower)s -v']
```

Comment:
This is probably not worth having, the 'files' sanity check should be enough.

Reply:
Actually no, I like having this, it rules out (simple) runtime issues...

We should do this more often, running simply commands like `--version` or `--help`...

So do leave as is @arielzn :)